relay/polkadot/tests/ahm/mod.rs

```diff
@@ -48,3 +49,101 @@ async fn remote_ext_test_setup() -> Option<RemoteExternalities<Block>> {
 
 	Some(ext)
 }
+
+#[test]
+fn call_filter_works() {
+	let mut t: sp_io::TestExternalities =
+		frame_system::GenesisConfig::<T>::default().build_storage().unwrap().into();
+
+	// MQ calls are never filtered:
```

Comment:
Did we get to the bottom of the prioritisation of system messages? I lost track of the status of your streaking work

Reply:
Yea it worked here but we should get it audited & merged https://github.com/paritytech/polkadot-sdk/pull/6059